<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChoirPage</class>
 <widget class="QWidget" name="ChoirPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>907</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font-family: &quot;Comic Sans MS&quot;;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_8">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QGroupBox" name="groupBox_leader">
       <property name="title">
        <string>Leader</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_17">
        <item>
         <widget class="QLabel" name="lbl_choir_leader">
          <property name="text">
           <string>leaderName</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_session">
       <property name="title">
        <string>Session</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="lbl_session_leader">
            <property name="text">
             <string>Leader:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_leader_name"/>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>48</width>
            <height>21</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="lbl_choir_volume">
            <property name="text">
             <string>Choir volume:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="img_session_volume">
            <property name="text">
             <string>volImg</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="horizontalSlider_session">
            <property name="styleSheet">
             <string notr="true">QSlider::groove:horizontal {
    border: 0px;
	background-color: rgb(48, 48, 48);
    height: 2px;
    margin: 0px;
    }
QSlider::handle:horizontal { 
	image: url(:/Images/Images/handle.png);
	border: 0px;
    height: 20px;
    width: 20px;
    margin: -15px 0px;
}</string>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="lbl_members">
            <property name="text">
             <string>Members:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbl_num_members">
            <property name="text">
             <string>Num</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="horizontalSpacer_9">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>18</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QGroupBox" name="groupBox_sopranos">
       <property name="title">
        <string>Sopranos</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QListView" name="listView_sopranos">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_altos">
       <property name="title">
        <string>Altos</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QListView" name="listView_altos">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_tenors">
       <property name="title">
        <string>Tenors</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QListView" name="listView_tenors">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_basses">
       <property name="title">
        <string>Basses</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QListView" name="listView_basses">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_16">
     <item>
      <widget class="QGroupBox" name="groupBox_you">
       <property name="title">
        <string>You</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="lbl_name">
            <property name="text">
             <string>Name:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_username"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_15">
          <item>
           <widget class="QLabel" name="lbl_section">
            <property name="text">
             <string>Section:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QComboBox" name="comboBox_section"/>
            </item>
            <item>
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_microphone">
       <property name="title">
        <string>Microphone</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_14">
          <item>
           <widget class="QLabel" name="img_microphone">
            <property name="text">
             <string>micImg</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QComboBox" name="comboBox_audio_inputs"/>
            </item>
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBar_microphone">
          <property name="styleSheet">
           <string notr="true">border: 0px ;
background-color: rgb(70, 215, 0);</string>
          </property>
          <property name="value">
           <number>24</number>
          </property>
          <property name="textVisible">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="img_microphone_volume">
            <property name="text">
             <string>speakImg</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="horizontalSlider_microphone">
            <property name="styleSheet">
             <string notr="true">QSlider::groove:horizontal {
    border: 0px;
	background-color: rgb(48, 48, 48);
    height: 2px;
    margin: 0px;
    }
QSlider::handle:horizontal { 
	image: url(:/Images/Images/handle.png);
	border: 0px;
    height: 20px;
    width: 20px;
    margin: -15px 0px;
}</string>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_headphones">
       <property name="title">
        <string>Headphones</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <widget class="QLabel" name="img_headphones">
            <property name="text">
             <string>setImg</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QComboBox" name="comboBox_audio_output"/>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QLabel" name="img_headphones_volume">
            <property name="text">
             <string>speakImg</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="horizontalSlider_headphones">
            <property name="styleSheet">
             <string notr="true">QSlider::groove:horizontal {
    border: 0px;
	background-color: rgb(48, 48, 48);
    height: 2px;
    margin: 0px;
    }
QSlider::handle:horizontal { 
	image: url(:/Images/Images/handle.png);
	border: 0px;
    height: 20px;
    width: 20px;
    margin: -15px 0px;
}</string>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>248</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
